src/server.rs

```diff
@@ -3,6 +3,9 @@ use crate::client::Client;
 use crate::replies::NumericReply;
 
 use std::collections::HashMap;
+use std::fs::File;
+use std::io::BufRead;
+use std::io::BufReader;
```

Comment:
This can be replaced with:
```suggestion
use std::io::{BufRead, BufReader};
```

Reply:
🤢